client/src/components/Button/Button.js

```diff
@@ -1,31 +1,31 @@
-import React from 'react';
-import { makeStyles } from '@material-ui/core/styles';
-import CircularProgress from '@material-ui/core/CircularProgress';
-import green from '@material-ui/core/colors/green';
-import MDButton from '@material-ui/core/Button';
-import CheckIcon from '@material-ui/icons/Check';
+import React from "react";
```

Comment:
single-quote, please.

Reply:
We haven't decided this to put a single quote or double quote and also es-lint is not giving any error so we can go with double quote